src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const costPerDay = 40;
```

Comment:
Instead of using the literal value 40, define a constant to explain its meaning. For example, 'const COST_PER_DAY = 40;'.